includes/User.php

```diff
@@ -2664,19 +2664,6 @@ public function getDefaultGlobalPreference($preference) {
 	}
 
 	/**
-	 * Get a user attribute local to this wikia.
-	 *
-	 * @param string $attr the attribute name
-	 * @param int $cityId the city id
-	 * @param string $sep the separator between the name and the city id
-	 * @return string
-	 * @see getGlobalAttribute for more documentation about attributes
-	 */
-	public function getLocalAttribute($attr, $cityId = null, $sep = "-") {
```

Comment:
This wasn't being used. All attributes we currently store are global, if we need to start supporting wiki-specific attributes down the line we can re-add this.